reana_cluster/configurations/reana-cluster-minikube-dev.yaml

```diff
@@ -11,6 +11,7 @@ cluster:
   shared_volume_path: "/var/reana"
   db_persistence_path: "/var/reana/db"
   debug: true
+  ui: true
```

Comment:
Needed?  If we use `--ui` in CLI technique then the same switch-on-switch-off technique could work with both PROD and DEV setups...